.github/workflows/qa.yml

```diff
@@ -1,7 +1,7 @@
 name: Q&A
 
 on:
-    pull_request:
+    pull_request_target:
```

Comment:
This makes the workflow checkout the `master` branch rather than the code of the PR, making the CI useless (as it does not test the PR)

Reply:
Oops, fixed in 8269d3d